Readme.md

```diff
@@ -27,6 +27,11 @@ This is a [Node.js](https://nodejs.org/en/) module available through the
 Before installing, [download and install Node.js](https://nodejs.org/en/download/).
 Node.js 0.10 or higher is required.
 
+If this is a brand new project, make sure to
+[create a package.json](https://docs.npmjs.com/creating-a-package-json-file)
+before installing express
+(suggestion: `$ npm init`)
```

Comment:
Would it be preferable to format the statement and command suggestion like the npm install below ?
- No $ sign
- Link to the npm docs for that command ?

Reply:
Right. I just resent a PR.